spec.md

```diff
@@ -92,39 +91,52 @@ The `Span` interface must have the following capabilities:
 
-The encoded data is separated in this way for a variety of reasons; the most important is to give OpenTracing users a portable way to opt out of Trace Attribute propagation entirely if they deem it a stability risk.
+#### Required Injection/Extraction formats
 
-The propagation and trace-joining methods come in two flavors: binary and text.
+At a minimum, all platforms are expected to require implementations to support two formats: the "split text" format and the "split binary" format. The "split" refers to two common components of propagated spans:
+
+1. The core identifying information for the `Span`, referred to as the "tracer state" (for example, in Dapper this would include a `trace_id`, a `span_id`, and a bitmask representing the sampling status for the given trace)
+1. Any trace attributes (per `Span`'s ability to set trace attributes that propagate across process boundaries)
+
+The "text" and "binary" designations refer to two flavors of encoding:
 
 - The *text* format is a platform-idiomatic map from (unicode) `string` to `string`; it is better-suited to pretty-printing and debugging
 - The *binary* format is an opaque byte array and is better-suited to compact, high-performance encoding, decoding, and transmission
 
-Note that there is no expectation that different tracing systems propagate `Spans` in compatible ways. Though OpenTracing is agnostic about the tracing implementation, for successful inter-process handoff it's essential that the processes on both sides of a propagation use the same tracing implementation.
+Note that there is no expectation that different tracing systems Inject and Extract `Spans` in compatible ways. Though OpenTracing is agnostic about the tracing implementation, for successful inter-process handoff it's essential that the processes on both sides of a propagation use the same tracing implementation.
+
+#### An end-to-end propagation example
+
+To make all the above more concrete, consider the following sequence:
+
+1. A client process has a `Span` instance and is about to make an RPC over a home-grown HTTP protocol
+1. That client process calls some `Tracer.Injector(...)` method, passing a format identifier for a string-to-string map
+1. With an `Injector` instance in hand, the client calls `injector.InjectSpan(...)`, passing the active `Span` instance and a string-to-string map carrier in as parameters
+1. The newly-populateed (opaque) string-to-string map is encoded into the homegrown HTTP protocol (e.g., as headers) by the application code
+1. Now in the server process, the application code grabs the opaque string-to-string map from the homegrown HTTP protocol
+1. The server process calls some `Tracer.Extractor(...)` method, again passing a format identifier for a string-to-string map
+1. With an `Extractor` instance in hand, the server calls `extractor.JoinTrace(...)`, passing in the desired operation name and string-to-string map carrier as parameters
```

Comment:
"string-to-string map carrier" sounds quite jargony.. if this is a forged example about http, would there be a way to say something more concrete?